doc/api/http.md

```diff
@@ -745,8 +745,16 @@ added: v0.1.90
 * `callback` {Function}
 
 Begin accepting connections on the specified `port` and `hostname`. If the
-`hostname` is omitted, the server will accept connections on any IPv6 address
-(`::`) when IPv6 is available, or any IPv4 address (`0.0.0.0`) otherwise.
+`hostname` is omitted, the server will accept connections on the [unspecified
+IPv6 address](https://en.wikipedia.org/wiki/IPv6_address#Unspecified_address)
+(`::`) when IPv6 is available, or the [unspecified IPv4 address
+](https://en.wikipedia.org/wiki/0.0.0.0) (`0.0.0.0`) otherwise.
+
+*Note*: in most operating systems, listening to the [unspecified
+IPv6 address](https://en.wikipedia.org/wiki/IPv6_address#Unspecified_address)
+(`::`) may cause the `net.Server` to also listen on the [unspecified IPv4 address
```

Comment:
Please move the actual link to the end and just use the `[unspecified IPv4 address][]` ref inline